pkg/interceptors/bitbucket/bitbucket.go

```diff
@@ -54,6 +54,10 @@ func (w *Interceptor) Process(ctx context.Context, r *triggersv1.InterceptorRequ
 	headers := interceptors.Canonical(r.Header)
 	// Validate secrets first before anything else, if set
 	if p.SecretRef != nil {
+		// Check the secret to see if it is empty
+		if p.SecretRef.SecretKey == "" {
+			return interceptors.Fail(codes.FailedPrecondition, "secret is empty")
```

Comment:
can we make this a bit more descriptive? e.g. interceptor secretRef.secretKey is empty?

Reply:
Will do, I'll make this update and push the changes. Thanks for the review.